src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -6072,21 +6070,21 @@ s32 func_8083EC18(Player* this, GlobalContext* globalCtx, u32 arg2) {
 
             if ((sp8C != 0) || (arg2 & 2) ||
                 func_80041E4C(&globalCtx->colCtx, this->actor.wallPoly, this->actor.wallPolySource)) {
-                f32 phi_f20;
+                s32 i;
                 CollisionPoly* sp84 = this->actor.wallPoly;
                 f32 sp80;
                 f32 sp7C;
                 f32 phi_f12;
                 f32 phi_f14;
+                Vec3f sp50[3];
+                f32 phi_f20;
```

Comment:
I imagine these stack changes aren't necessary, could they be omitted if possible?